i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md

```diff
@@ -2,29 +2,44 @@
 sidebar_position: 3
 sidebar_class_name: sidebar-item-getting-started
 sidebar_label: まずはじめに
-title: Momento サーバーレスキャッシュサービスをはじめてみる
-description: SDK またはCLI を使って、Momento サーバーレスキャッシュをすぐに使い始めてみましょう。
+title: Momento Serverless Cache をはじめてみる
+description: SDK またはCLI を使って、Momento Serverless Cache をすぐに使い始めてみましょう。
 ---
 
-# Momento サーバーレスキャッシュをはじめてみる
+# Momento Serverless Cache をはじめてみる
 
-SDK または CLI を使って Momento サーバーレスキャッシュをすぐに使い始める方法をお探しでしたら、ここから始めましょう。サービスにアクセスする方法は以下の様に複数あります。
+SDK または CLI を使って Momento Serverless Cache をすぐに使い始める方法をお探しでしたら、ここから始めましょう。ソフトウェアのインストールを全くせずにMomento をすぐに体験してみたいですか？でしたら、[ブラウザを使って](#cli-とsdk-をブラウザ上で試してみる)ここからすぐに始められます。もしくは、既に開発環境を整えてMomento を使ったアプリケーションを使い始める準備ができている場合は、[以下のシンプルな手順](#momento-command-line-interface-cli-をインストールする)に従って下さい。
 
-## CLI をブラウザ上で試してみる
+## CLI とSDK をブラウザ上で試してみる
 
-私たちの CLI ツールをブラウザ上で今試してみることができます。何のソフトウェアもインストールする必要はありません！
+ブラウザ上で、今すぐにMomento Serverless Cache を試してみることができます。何もソフトウェアをインストールする必要がありません！
 
-[Momento サーバーレスキャッシュを試してみる](https://www.gomomento.com/try-momento-for-free)に行って、
-ブラウザ内の shell セッションで直接 Momento CLI のコマンドを実験してみることができます。
-無料の認証トークンを作成し、キャッシュを作成し、そのキャッシュ上で`set` や`get` コマンドを実行してみることができます。
-試してみるのに数分もかかりません！
+:::note
+Momento Serverless Cache はAPI ベースのサーバーレスサービスです。皆さんの(AWS、GCP、Azure、等)のアカウントには何もデプロイしません。
+:::
+
+<br />
+
+まずはじめに、無料の認証トークンを作成し、キャッシュを作成し、そのキャッシュ上で`set` や`get` コマンドを実行してみましょう。以下を起動します。
+
+<a href="https://play.instruqt.com/embed/momento/tracks/sandbox-container-1challenge?token=em_14J4EucaTHzL0rXw&finish_btn_target=_top&finish_btn_text=Return+to+Docs&finish_btn_url=https%3A%2F%2Fdocs.momentohq.com%2Fgetting-started#try-our-cli-and-an-sdk-in-your-browser" target="_top"><img src="/img/cli_lab.png" alt="CLI lab" /></a>
+
+<br />
+<br />
+
+続いて、Node.js SDK を使ったJavaScript のアプリケーションを動かして、認証トークンと今作成したキャッシュを使ってみましょう。以下を起動します。
+
+<a href="https://play.instruqt.com/embed/momento/tracks/momento-nodejs-demo?token=em_f8PM8Aob-mHIfOTT&finish_btn_target=_top&finish_btn_text=Return+to+Docs&finish_btn_url=https%3A%2F%2Fdocs.momentohq.com%2Fgetting-started#try-our-cli-and-an-sdk-in-your-browser" target="_top"><img src="/img/sdk_lab.png" alt="SDK lab" /></a>
```

Comment:
Could you update the image source here as well?
<img width="815" alt="image" src="https://user-images.githubusercontent.com/22686260/218360415-fafebde7-7e81-4e53-b6ee-b9d5b69f637b.png">

Reply:
Updated. The image must be stored under `i18n/ja` instead of `static`. I left the original version as is. It might be good to align with it, though.

https://github.com/momentohq/public-dev-docs/compare/22ff04d3a0ecdf734207a6f2f4474a87551fca26..2c20a140e9b888b43ebe5e8d6f20ae09b2b0596f